### ISSUE TITLE
Enable building with useFrameworks: "static" for iOS devices

### ISSUE THREAD

**Issue by MarchewkaMatthew** [allboatsrise/expo-marketingcloudsdk#11]
Hi @andrejpavlovic !

Thank you for maintaining this module. I have a question regarding building this plugin with `useFrameworks: "static"` set for iOS devices (added with the `expo-build-properties`). I need to add it in order to enable `React Native Firebase` in my project. I'm not really sure what that would require from the package to support building from the "framework" instead of the static library.

`app.config.ts`:
```
      [
        "expo-build-properties",
        {
          ios: {
            useFrameworks: "static",
          },
        },
      ],
```

Error message during `run fastlane` step in the eas build:

<img width="842" alt="image" src="https://github.com/allboatsrise/expo-marketingcloudsdk/assets/56259093/d5975d69-6937-4843-93b9-8a594627fd93">

Thank you in advance for your help!

**Comment by andrejpavlovic** [comment]
Isn't that error related to this issue?
https://github.com/allboatsrise/expo-marketingcloudsdk/issues/6

**Comment by MarchewkaMatthew** [comment]
Hi @andrejpavlovic, I'm not sure. in the eas build steps `expo prebuild` seems to work just fine. I also have `expo-notifications` added to my plugins list. I will try to add `s.dependency 'EXNotifications'` and let you know about the result.

**Comment by MarchewkaMatthew** [comment]
It indeed fixed the issue 👍  I've updated my PR to this repository: #9

**Comment by andrejpavlovic** [comment]
Published fix in `48.0.1`
